crates/matrix-sdk-crypto/src/session_manager/sessions.rs

```diff
@@ -257,43 +260,70 @@ impl SessionManager {
                     let is_timed_out = self.is_user_timed_out(&user_id, &device_id);
 
                     if is_missing && is_timed_out {
-                        timed_out.entry(user_id.to_owned()).or_default().insert(device_id);
+                        timed_out_devices_by_user
+                            .entry(user_id.to_owned())
+                            .or_default()
+                            .insert(device_id);
                     } else if is_missing && !is_timed_out {
-                        missing
+                        missing_session_devices_by_user
                             .entry(user_id.to_owned())
                             .or_default()
-                            .insert(device_id, DeviceKeyAlgorithm::SignedCurve25519);
+                            .insert(device_id);
```

Comment:
partly another rename, but also, rather than building a map of devices where every value is `SignedCurve25519`, we just build a set for now, which makes the value clearer when we log it.

Reply:
Jonas taught me about [`Itertools::format`](https://docs.rs/itertools/latest/itertools/trait.Itertools.html#method.format) today, which could be useful here: we could keep the previous format (and avoid reconstructing the expected type before sending the request at the end of this function), hiding the algorithm by using iterator combinators and then formatting the resulting iterator in the log line. (We're already using `itertools` in this crate.)

Reply:
ahh, that's a handy technique! Addressed in 2a2dd512c.